src/main.js

```diff
@@ -9,70 +8,94 @@ var btnMakeNewCover = document.querySelector(".make-new-button");
 var btnViewSavedCover = document.querySelector(".view-saved-button");
 var btnHome = document.querySelector(".home-button");
 var btnSaveCover = document.querySelector(".save-cover-button");
-var btnMakeMyBook = document.querySelector(".create-new-book-button");//3rd iteration
+var btnMakeMyBook = document.querySelector(".create-new-book-button");
 var homeView = document.querySelector(".home-view");
 var savedView = document.querySelector(".saved-view");
 var formView = document.querySelector(".form-view");
-var inputCover = document.querySelector(".user-cover");///vars below will be on Jessica's branch
+var inputCover = document.querySelector(".user-cover")
 var inputTitle = document.querySelector(".user-title");
 var inputDescriptor1 = document.querySelector(".user-desc1");
 var inputDescriptor2 = document.querySelector(".user-desc2");
 var form = document.querySelector("form");
 
 var savedCovers = [
-  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
+  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
```

Comment:
not sure if this will be taken out later and if you're just using it as a test for now but, if you plan to keep this instance, i'd recommend breaking all the arguments into separate variables just to reduce the line length

Reply:
This might be a part of our last Iteration. This piece of code was provided for us, we we haven't deleted it or used it yet.